database/tls_test.go

```diff
@@ -0,0 +1,27 @@
+package database_test
+
+import (
+	"testing"
+
+	"github.com/moov-io/base/database"
+	"github.com/moov-io/base/log"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_LoadClientCertsFromConfig(t *testing.T) {
+
+	config := &database.MySQLConfig{
+		TLSClientCerts: []database.TLSClientCertConfig{
+			{
+				CertFilePath: "testdata/client_cert.pem",
+				KeyFilePath:  "testdata/client_cert_private_key.pem",
```

Comment:
These are missing, but can we build the paths in an OS-agnostic way?

```suggestion
				CertFilePath: filepath.Join("testdata", "client_cert.pem"),
				KeyFilePath:  filepath.Join("testdata", "client_cert_private_key.pem"),
```